.gitignore

```diff
@@ -133,9 +109,26 @@ cmake_install.cmake
 /addons/adsp.*
 /addons/peripheral.*
 /addons/game.*
+/addons/kodi.binary.global.audioengine/addon.xml
+/addons/kodi.binary.global.filesystem/addon.xml
+/addons/kodi.binary.global.general/addon.xml
+/addons/kodi.binary.global.gui/addon.xml
+/addons/kodi.binary.global.main/addon.xml
+/addons/kodi.binary.global.network/addon.xml
+/addons/kodi.binary.instance.adsp/addon.xml
+/addons/kodi.binary.instance.audiodecoder/addon.xml
+/addons/kodi.binary.instance.audioencoder/addon.xml
+/addons/kodi.binary.instance.game/addon.xml
+/addons/kodi.binary.instance.imagedecoder/addon.xml
+/addons/kodi.binary.instance.inputstream/addon.xml
+/addons/kodi.binary.instance.peripheral/addon.xml
+/addons/kodi.binary.instance.pvr/addon.xml
+/addons/kodi.binary.instance.screensaver/addon.xml
+/addons/kodi.binary.instance.vfs/addon.xml
+/addons/kodi.binary.instance.videocodec/addon.xml
+/addons/kodi.binary.instance.visualization/addon.xml
```

Comment:
Oh right, I symlink /build/addons to /addons to avoid having to recompile binary add-ons (a big pain with 60 game add-ons) whenever I remove and recreate my build folder